sign/sign.go

```diff
@@ -76,6 +80,12 @@ func Sign(ctx context.Context, opts *Options) error {
 		cmd.Args = append(cmd.Args, "--entitlements", opts.Entitlements)
 	}
 
+	if len(opts.Requirements) > 0 {
+		//requirementsString := fmt.Sprintf("-r=\"%s\"", opts.Requirements)
+		requirementsString := fmt.Sprintf("-r=%s", opts.Requirements)
```

Comment:
Would `%q` be more appropriate?

Reply:
Likely.
I'll look at what %q does, I'm still understanding all the format strings

Reply:
I'm just a nobody that happened to look at your PR. Glad that works out.